### ISSUE TITLE
How to use a @tensorflow-models model from python's tensorflow?

### ISSUE THREAD

**Issue by galah92** [tensorflow/tfjs#2901]
Hi,

I'm developing a project in "regular" TensorFlow (over PC and Colab) with embedded usage scenario.
I'd like to use the handpose model ([introduced here](https://blog.tensorflow.org/2020/03/face-and-hand-tracking-in-browser-with-mediapipe-and-tensorflowjs.html)) to generate labels for my project. Is there a way to use that model from python's TensorFlow?

I though converting to model to Keras `SavelModel` (or even `tflite`), but I can't even find the tfjs model weights to be converted. What am I missing?

Thanks

**Comment by rthadur** [comment]
Closing this due to lack of activity, feel to reopen. Thank you
